src/seedu/addressbook/commands/CommandResult.java

```diff
@@ -33,4 +33,13 @@ public CommandResult(String feedbackToUser, List<? extends ReadOnlyPerson> relev
         return Optional.ofNullable(relevantPersons);
     }
 
+    /**
+     * 
+     * @return the private attribute feedbackToUser
```

Comment:
If the method is too simple, you can write the method description instead of using `@return`, e.g. `Returns ...`.

BTW, "the private attribute feedbackToUser" is not a good comment.